Maps.md

```diff
@@ -0,0 +1,25 @@
+This is a general guide about Maps for all FullScreenShenanigans projects. This guide will explain the concept of Maps, Areas, and Locations within a game and how they're connected.
+
+### Table of Contents
+1. [MapsCreatr](#mapscreatr)
+2. [AreaSpawnr](#areaspawnr)
+
+# MapsCreatr
+
+MapsCreatr is a module that handles the storage of Maps, Areas, and Locations. 
+
+A Map is a collection of Areas and an Area is a space with predefined boundaries and Things. Locations are entry points within an Area, where an entry point is where the player ends up after transitioning to a new Area.
+
+Generally in a FullScreenShenanigans project, each Map information is stored in its own .js file, outlining all Areas and Locations within that map. When the game is started, MapsCreatr loads all this data, goes through every Map and uses ObjectMakr to create a new Map object, and finally stores it for later reference.
+
+More can be read about MapsCreatr on its [Readme](https://github.com/FullScreenShenanigans/MapsCreatr/blob/master/README.md).
+
+# AreaSpawnr
+
+AreaSpawnr is a module that works with MapsCreatr to spawn Things in a certain Area.
+
+When a new Area is entered by the player, AreaSpawnr uses the information from that Area's Map, stored in MapsCreatr, to spawn that Area's predefined Things (or PreThings) in the Area. 
```

Comment:
Add more specificity here. How does it use the information from the map? How does it know which things to spawn, and where? What triggers this?